app/services/watsonAssistant.js

```diff
@@ -14,7 +14,7 @@ module.exports = function() {
         assistantId: process.env.WATSON_ASSISSTANT_ID,
       })
           .then((res) => {
-            console.log(JSON.stringify(res, null, 2));
+            // console.log(JSON.stringify(res, null, 2));
```

Comment:
Console-Statements sollten grundsätzlich nicht in `dev` gepusht werden, aber an dieser Stelle kann das vernachlässigt werden.

Reply:
Es hat mich hier nur genervt beim debuggen :D